packages/client-core/src/admin/components/moderation/ModerationBanContainer.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { API, useFind, useMutation } from '@ir-engine/common'
+import { moderationBanPath, ModerationBanType } from '@ir-engine/common/src/schema.type.module'
+import { AbuseReasonsType } from '@ir-engine/common/src/schemas/moderation/moderation.schema'
```

Comment:
@HexaField what should be the practice for AbuseReasonsType as its an array which types of string.

Reply:
Hmm i'm not quite sure what you are asking

Reply:
This is fine. Just need to make its cast "as const" such that it isn't just an array of strings but an array of string literals